safe_eth/eth/account_abstraction/bundler_client.py

```diff
@@ -172,14 +178,21 @@ def get_user_operation_and_receipt(
             self._get_user_operation_by_hash_payload(user_operation_hash, request_id=1),
             self._get_user_operation_receipt_payload(user_operation_hash, request_id=2),
         ]
-        result = self._do_request(payload)
-        if not (result and result[0]):
+        results = self._do_request(payload)
+        if (
+            results
+            and isinstance(results, list)
+            and results[0] is not None
+            and isinstance(results[0], dict)
+            and results[1] is not None
+            and isinstance(results[1], dict)
+        ):
+            return UserOperation.from_bundler_response(
+                user_operation_hash, results[0]
+            ), self._parse_user_operation_receipt(results[1])
+        else:
```

Comment:
`else` is not required after `return`

Reply:
Fixed [here](https://github.com/safe-global/safe-eth-py/pull/1308/commits/1d71d495526362616735c2fdb81c507054333525)